app/src/parser_print_txn.c

```diff
@@ -79,7 +79,7 @@ __Z_INLINE parser_error_t printFee(const parser_context_t *ctx, char *outKey, ui
     return parser_ok;
 }
 
-static parser_error_t printBondTxn( const parser_context_t *ctx,
+static __attribute__((noinline)) parser_error_t printBondTxn( const parser_context_t *ctx,
```

Comment:
i think that a macro in zxlib could simplify this:
https://github.com/Zondax/ledger-zxlib/blob/main/include/zxmacros.h#L40

Reply:
The macro is for inlining we are removing the inlining. For this bigger functions, it seems to help reduce stack usage.